user/python-gbinder/patches/cython.patch

```diff
@@ -0,0 +1,35 @@
+--- a/setup.py	2023-09-16 14:58:25.000000000 -0300
++++ b/setup.py	2024-09-07 12:26:23.808413594 -0300
+@@ -1,5 +1,6 @@
+ import sys, subprocess
+ from distutils.core import setup, Extension
++from Cython.Build import cythonize
+ 
+ 
+ def pkgconfig(package, kw):
+@@ -10,23 +11,12 @@
+         kw.setdefault(flag_map.get(token[:2]), []).append(token[2:])
+     return kw
+ 
+-USE_CYTHON = False
```

Comment:
this patch is one line if you change this False to True and make literally no other changes